ui/conf/nginx.conf

```diff
@@ -2,8 +2,23 @@ server {
     listen       80;
     server_name  localhost;
 
-    #charset koi8-r;
-    #access_log  /var/log/nginx/host.access.log  main;
+    # https://nginx.org/en/docs/http/ngx_http_gzip_module.html
+    gzip on;
+    gzip_static on;
+    gzip_comp_level    5;
+    gzip_min_length    256;
+
+    gzip_types
+    application/javascript
```

Comment:
Consider indenting these a bit more, for clarity.

Reply:
Yep, I just add an indent.